mkdocs.yml

```
@@ -1,21 +1,66 @@
-site_name: ACMBPDC
+site_name: ACM Coding Bootcamp
+docs_dir: './docs'
+remote_branch: mkdocs-webpage
```

Could you change the `remote_branch` to `gh-pages`
```suggestion
remote_branch: gh-pages
```